projects/project-player-library/src/lib/constants/actionConstants.ts

```diff
@@ -33,7 +39,7 @@ export const actions = {
     },
     DOWNLOADED_ACTION : {
         title: "DOWNLOADED",
-        icon: "check_circle",
+        icon: "cloud_download",
```

Comment:
change to icon to the previous one
"check_circle"

Reply:
okay